includes/options/class-amp-options-manager.php

```diff
@@ -33,6 +33,16 @@ class AMP_Options_Manager {
 		Option::VERSION                 => AMP__VERSION,
 	];
 
+	/**
+	 * Sets up hooks.
+	 */
+	public static function init() {
```

Comment:
Broke these hooks into a separate method because `register_settings` needs to run on `rest_api_init` as well as `admin_init`, while these just need to run on `admin_init`